docs/src/main/sphinx/installation/cli.rst

```diff
@@ -48,15 +48,59 @@ recognizes these certificates.
 
 Authentication
 --------------
+Password/LDAP Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
 password value to avoid the prompt.
 
+When Trino is configured with LDAP authentication, invoking CLI requires additional options
+such as ``--keystore-*`` or ``--truststore-*`` to secure TLS connection.
+The simplest way to invoke the CLI is with a wrapper script.
+
+.. code-block:: text
+
+    #!/bin/bash
```

Comment:
This could be used as an advanced example but probably not in the ldap/password auth section since it does too much at once